api/models/cartDao.js

```diff
@@ -0,0 +1,60 @@
+const dbDataSource = require("./dataSource");
+
+const postCart = async (condition) => {
+  // cart에 수량이랑 user_id랑 다 같이 넘거야함
+  let productId = condition.productId;
+  let size = condition.size;
+  let color = condition.color;
+  let quantity = condition.quantity;
+
+  if (color == "blue") {
+    color == 1;
+  } else if (color == "black") {
+    color == 2;
+  } else if (color == "gray") {
+    color == 3;
+  } else if (color == "red") {
+    color == 4;
+  } else if (color == "yellow") {
+    color == 5;
+  } else if (color == "green") {
+    color == 6;
+  } else if (color == "purple") {
+    color == 7;
+  }
```

Comment:
if-else보다는 아래와 같이 객체를 만들어서 사용하는 것이 훨씬 좋을 것 같습니다.
size도 다음과 같이 수정해 주세요.
```suggestion
const colors =  {
  blue: 1,
  black: 2,
  gray: 3,
  red: 4,
  yellow: 5,
  green: 6,
  purple: 7
}
```